### DISCUSSION TITLE
Failed on Install Plex LXC (multiple tried)

### DISCUSSION THREAD

**Discussion by kiraje** [tteck/Proxmox#3153] [Help Needed]
____  __             __  ___         ___          _____                          
   / __ \/ /__  _  __   /  |/  /__  ____/ (_)___ _   / ___/___  ______   _____  _____
  / /_/ / / _ \| |/_/  / /|_/ / _ \/ __  / / __ `/   \__ \/ _ \/ ___/ | / / _ \/ ___/
 / ____/ /  __/>  <   / /  / /  __/ /_/ / / /_/ /   ___/ /  __/ /   | |/ /  __/ /    
/_/   /_/\___/_/|_|  /_/  /_/\___/\__,_/_/\__,_/   /____/\___/_/    |___/\___/_/     
                                                                                      
Using Default Settings
Using Distribution: ubuntu
Using ubuntu Version: 22.04
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 100
Using Hostname: plex
Using Disk Size: 8GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Plex LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 \update failed - see /var/log/pveam.log for details
 ✓ Updated LXC Template List
 \^Cdownload failed: got interrupted by signal
 ✓ Downloaded LXC Template
 \unable to create CT 100 - volume 'local:vztmpl/ubuntu-22.04-standard_22.04-1_amd64.tar.zst' does not exist
bash: line 182: exit: A problem occured while trying to create container.: numeric argument required

**Comment by tteck** [comment]
> ^Cdownload failed: got interrupted by signal

Why are you doing a Ctrl+c ?

> ubuntu-22.04-standard_22.04-1_amd64.tar.zst' does not exist

The template does not exist because it was never downloaded

**Reply by kiraje** [reply]
Uh, lmc again :|

**Reply by kiraje** [reply]
Sry about this low tech
Have just know about this self host few days back

**Comment by SJRM78** [comment]
Just to comment, i had the same issue, i was not able to create any CT container.

the issue was with my **/etc/subgid** and **/etc/subuid** filles  (i was trying to make id mapping to work).
I rolled back those files to the originals, and now it works
